src/client/clientmap.cpp

```diff
@@ -124,12 +124,15 @@ void ClientMap::updateDrawList()
 
 	v3f camera_position = m_camera_position;
 	v3f camera_direction = m_camera_direction;
+	f32 camera_fov = m_camera_fov;
```

Comment:
does this actually make a difference?

Reply:
It's a pattern used elsewhere. I'm pretty sure it does not make a difference, but wanted to follow the pattern.